src/components/dashboard/SendLinkSummary.js

```diff
@@ -267,6 +270,42 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
     }
   }, [amount, counterPartyDisplayName, fullName, navigateTo, getLink])
 
+  const sendViaBridge = useCallback(
+    async (amount: string) => {
+      await sendBridgeRequest(amount, network.toLowerCase())
+    },
+    [amount],
+  )
+
+  useEffect(() => {
+    const handleBridgeTransaction = () => {
+      const { transaction: tx } = bridgeRequestStatus
+      const transactionEvent: TransactionEvent = {
+        id: tx.hash,
+        createdDate: new Date().toISOString(),
+        date: new Date().toISOString(),
+        type: FeedItemType.EVENT_TYPE_SENDDIRECT,
+        chainId: goodWallet.networkId,
+        data: {
+          to: goodWallet.account,
+          reason: 'Bridging',
+          amount,
+        },
+      }
+
+      log.debug('sendViaAddress: enqueueTX', { transactionEvent })
+
+      userStorage.enqueueTX(transactionEvent)
+
+      fireEvent(BRIDGE_INITIATED, { type: 'Bridge' })
```

Comment:
the event should be just like any TX. the type is the diffrentiator

Reply:
done